components/buildless-serverless/PROJECT

```diff
@@ -0,0 +1,20 @@
+# Code generated by tool. DO NOT EDIT.
```

Comment:
which tool is consuming this file? maybe we can get rid of ti?

Reply:
Thanks for that, there is no tool which consumes this file, I've deleted it